website_snippet_country_dropdown/views/snippets.xml

```diff
@@ -52,13 +51,14 @@
 
 <template id="country_dropdown" name="Country Dropdown">
     <t t-set="countries" t-value="request.env['res.country'].search([])"/>
-    <div id="snippet_container" class="input-group" t-ignore="true">
-        <div class="input-group-btn">
+    <t t-set="default_country" t-value="countries[:1]"/>
```

Comment:
I have reviewed https://github.com/OCA/e-commerce/pull/110 and I still think you should do https://github.com/OCA/website/pull/210#discussion_r69418026. After all, it's a 1-line change, less than adding something to the roadmap.